src/mobile/components/article/templates/sections/slideshow.jade

```diff
@@ -6,11 +6,9 @@ if section.items.length > 0
     - item = section.items[0]
     case item.type
       when 'image'
```

Comment:
I'm curious as to why this wouldn't need to change to `image_collection` as well like in the `/article/templates/index.jade` file?

Reply:
The slideshow is a section type, it is a sibling of of `image_collection` -- here we are rendering individual items in a slideshow, it mimics behavior of image_collection but with the added ability to render a video.